charts/prometheus/values.yaml

```diff
@@ -696,6 +696,13 @@ server:
   ### When empty server.persistentVolume.mountPath is used instead
   storagePath: ""
 
+  ## If true, binds the Prometheus server to localhost and hence does not accept traffic from outside the
+  ## Kubernetes Pod. Useful in service mesh setups (e.g. Consul Connect, Istio, ...), where traffic flows through
+  ## sidecar proxies.
+  ## If using that feature, you most likely also need to configure "server.customReadinessProbe" and "server.customLivenessProbe"
+  ## and expose the respective HTTP paths on different ports outside of the service mesh (e.g. via Consul's exposed paths feature)
+  listenOnLocalhost: false
```

Comment:
Imho if we make use of "--web.listen-address" it should be configurable and not bound to localhost only.